### PR TITLE
add the cobra command for mayactl volume create

### PR DESCRIPTION
**Pull request by kmova**
**What this PR does / why we need it**:
This is part of the maya code base cosolidation (and cleanup) effort to make use of similar packages for CLI across all maya components. Refer: https://github.com/openebs/openebs/issues/374 https://github.com/openebs/openebs/issues/388

This PR builds on the changes made in:
https://github.com/openebs/maya/pull/131 (using the mapiserver client)
https://github.com/openebs/maya/pull/135 (removing the unused logging packages)

This PR is an incremental re-factor to fix the issue : https://github.com/openebs/openebs/issues/505

**Special notes for your reviewer**:
- Following the removal of logging packages as part of PR #135, vendor and Godep files have been updated. 
- command/volume_create has been refactored to delegate the task for making the rest call for volume create to pkg/client/mapiserver
- cmd/mayactl/app/command -- has been updated with a COBRA command for volume create

### CODE REVIEW

File: command/volume_create.go [hunk 1]
```diff
@@ -106,69 +100,12 @@ func (c *VsmCreateCommand) Run(args []string) int {
 			return 1
 		}
 
-		resp := CreateAPIVsm(c.vsmname, c.size)
+		resp := mapiserver.CreateVolume(c.vsmname, c.size)
 		if resp != nil {
-			c.Ui.Error(fmt.Sprintf("Error Creating Volume %v", resp))
+			c.Ui.Error(fmt.Sprintf("Error Creating Volume: %v", resp))
```
Comment (AmitKumarDas) [inline command/volume_create.go]: Does this not belong to old CLI framework ?

File: cmd/mayactl/app/command/commands.go [hunk 2]
```diff
@@ -25,12 +25,12 @@ func NewMayaCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "maya",
```
Comment (AmitKumarDas) [inline cmd/mayactl/app/command/commands.go]: Should these files be under any version ?

File: command/volume_create.go (hunk 1 above)
Comment (kmova) [reply command/volume_create.go]: this file still is not removed. will remove all the files under command/.. once the COBRA cli is functional

File: cmd/mayactl/app/command/commands.go (hunk 2 above)
Comment (kmova) [reply cmd/mayactl/app/command/commands.go]: probably not.. but the type these commands use to interact with mapiserver etc., should be versioned.

File: command/volume_create.go (hunk 1 above)
Comment (AmitKumarDas) [reply command/volume_create.go]: ok

**Pull request merged**